### PR TITLE
remove execution time provisioning

### PR DESCRIPTION
**Pull request by javanlacerda**

### CODE REVIEW

File: kubejobs/__init__.py [hunk 1]
```diff
@@ -548,8 +534,7 @@ def execute(self, data):
 
 def rebuild(app_id, starting_time,
```
Comment (matheuscmelo) [inline kubejobs/__init__.py]: As you changed the parameters of this function, you need to change any calls to this function too, like this one
https://github.com/ufcg-lsd/asperathos-manager/blob/594ef071e3407c2b1c982fc57aceec2a779c4c86/kubejobs/__init__.py#L118-L131

File: kubejobs/__init__.py [hunk 2]
```diff
@@ -79,7 +77,6 @@ def __repr__(self):
             "starting_time": str(self.get_application_start_time()),
             "status": self.status,
             "visualizer_url": self.visualizer_url,
-            "execution_time": self.execution_time,
```
Comment (matheuscmelo) [inline kubejobs/__init__.py]: As you're removing this attribute from the Job entity, shouldn't you update the docs for this? https://github.com/ufcg-lsd/asperathos-manager/blob/master/docs/restapi-endpoints.md

**Review by matheuscmelo — CHANGES_REQUESTED** [review]
Some minor changes and questions.

File: kubejobs/__init__.py (hunk 2 above)
Comment (joseims) [reply kubejobs/__init__.py]: Agreed

**Pull request merged**